linkerd/protocol/http/src/main/scala/io/buoyant/linkerd/protocol/http/ErrorResponder.scala

```diff
@@ -23,12 +23,18 @@ class ErrorResponder extends SimpleFilter[Request, Response] {
         case ErrorResponder.HttpResponseException(rsp) =>
           rsp
         case _ =>
-          log.error(e, "service failure")
           val message = e.getMessage match {
             case null => e.getClass.getName
             case msg => msg
           }
-          val rsp = Headers.Err.respond(message, Status.BadGateway)
+          val status = e match {
+            case _: TimeoutException | Failure(Some(_: TimeoutException)) =>
+              Status.ServiceUnavailable
```

Comment:
I was following the pattern from here: https://github.com/twitter/finagle/blob/72cac55fecfd13daf00a4cfed4dad11b926baa8d/finagle-core/src/main/scala/com/twitter/finagle/service/RetryPolicy.scala#L169